ct-app/core/subgraph/graphql_provider.py

```
@@ -47,7 +47,7 @@ def _initialize_query(
         if self.default_key is None:
             self.default_key = keys
 
-    def _load_query(self, path: Union[str, Path], extra_inputs: list[str] = []) -> str:
+    def _load_query(self, path: str | Path, extra_inputs: list[str] = []) -> str:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)